featurebyte/schema/historical_feature_table.py

```diff
@@ -37,7 +37,7 @@ class HistoricalFeatureTableListRecord(BaseMaterializedTableListRecord):
     """
 
     feature_store_id: PydanticObjectId
-    observation_table_id: PydanticObjectId
+    observation_table_id: Optional[PydanticObjectId]
```

Comment:
Would be good to mention this fix in the changelog as well. Do we need to backport this to 0.4?